src/main/java/com/julienvey/trello/Trello.java

```diff
@@ -79,6 +79,10 @@
 
     TList getList(String listId, Argument... args);
 
+    /* CheckLists */
+
+    CheckList getCheckList(String checkListId, Argument... args);
```

Comment:
Actually, I'm confused here, not sure if the checklist is always associated with a card.

Might be ok like it is now